frontend_lib/src/component/Timeline/CommentFilePreview.jsx

```diff
@@ -9,82 +11,113 @@ import {
-      <div
-        className={classnames(`${props.customClass}`, 'comment', {
-          sent: fromMe,
-          received: !fromMe
-        })}
-        style={fromMe ? styleSent : {}}
-      >
+    return (
+      <li className={classnames(`${props.customClass}__messagelist__item`, 'timeline__messagelist__item')}>
         <div
-          className={classnames(`${props.customClass}__body`, 'comment__body')}
+          className={classnames(`${props.customClass}`, 'comment', {
+            sent: fromMe,
+            received: !fromMe
+          })}
+          style={fromMe ? styleSent : {}}
         >
-          <div className='comment__body__content'>
-            <Avatar
-              size={AVATAR_SIZE.MEDIUM}
-              user={apiAuthor}
-              apiUrl={props.apiUrl}
-            />
-            <div className='comment__body__content__text'>
-              <div className={classnames(`${props.customClass}__body__author`, 'comment__body__author')}>
-                {apiAuthor.public_name}
-              </div>
+          <div
+            className={classnames(`${props.customClass}__body`, 'comment__body')}
+          >
+            <div className='comment__body__content'>
+              <Avatar
+                size={AVATAR_SIZE.MEDIUM}
+                user={apiAuthor}
+                apiUrl={props.apiUrl}
+              />
+              <div className='comment__body__content__text'>
+                <div className={classnames(`${props.customClass}__body__author`, 'comment__body__author')}>
+                  {apiAuthor.public_name}
+                </div>
 
-              <div
-                className={classnames(`${props.customClass}__body__date`, 'comment__body__date')}
-                title={createdFormatted}
-              >
-                {props.apiContent.created}
-              </div>
+                <div
+                  className={classnames(`${props.customClass}__body__date`, 'comment__body__date')}
+                  title={createdFormatted}
+                >
+                  {props.apiContent.created}
+                </div>
 
-              <div
-                className={classnames(`${props.customClass}__body__text`, 'comment__body__text')}
-              >
-                <img
-                  className={classnames(`${props.customClass}__body__text__asFile`, 'comment__body__text__asFile')}
-                  src={previewUrl}
-                  alt={props.apiContent.filename}
-                />
+                <div
+                  className={classnames(`${props.customClass}__body__text`, 'comment__body__text')}
+                  title={title}
+                >
+                  {(this.state.fallbackPreview
+                    ? (
+                      <>
+                        <Icon icon='fas fa-fw fa-paperclip' title='' />
+                        {' '}
+                        {filename}
```

Comment:
Just a suggestion, you can resolve the comment if you don't want to apply
```suggestion
                        <Icon icon='fas fa-fw fa-paperclip' title='filename' />
                        {` ${filename}`}
```

Reply:
I would not put a title there because the parent already has a title, but I can apply the second line.
```suggestion
                        <Icon icon='fas fa-fw fa-paperclip' title='' />
                        {` ${filename}`}
```